coloring.dylan

```diff
@@ -0,0 +1,39 @@
+Module: %testworks
+Synopsis: Utilities and code that needs to be loaded early.
+Copyright: Original Code is Copyright (c) 1995-2004 Functional Objects, Inc.
+           All rights reserved.
+License: See License.txt in this distribution for details.
+Warranty: Distributed WITHOUT WARRANTY OF ANY KIND
+
+
+define constant $passed-attributes
+  = text-attributes(foreground: $color-green);
+define constant $skipped-attributes
+  = text-attributes(foreground: $color-cyan);
+define constant $not-implemented-attributes
+  = text-attributes(foreground: $color-cyan);
+define constant $failed-attributes
+  = text-attributes(foreground: $color-red);
+define constant $crashed-attributes
+  = text-attributes(foreground: $color-red);
+define constant $component-name-attributes
+  = text-attributes(intensity: $bright-intensity);
+define constant $total-attributes
+  = text-attributes(intensity: $bright-intensity);
+define constant $count-attributes
+  = text-attributes(intensity: $bright-intensity);
+
+define function result-status->attributes
```

Comment:
The arrow notation isn't very common. I'd probably just call this result-status-attrubites.

.oO( Would result-status→attributes blow up? )